CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [BUGFIX] Alertmanager: fix multitenant status page if clustering via gossip is disabled or sharding is enabled. #4184
```

Comment:
[nit] Please move this CHANGELOG entry below, along with the group of `[BUGFIX]` entries. We keep them grouped together by type.